crowbar_framework/app/models/service_object.rb

```diff
@@ -254,6 +249,22 @@ def remove_pending_elements(bc, inst, elements)
     end
   end
 
+  def set_to_applying(nodes, inst)
+    f = acquire_lock "BA-LOCK"
+    begin
+      nodes.each do |node_name|
+        node = NodeObject.find_node_by_name(node_name)
+        next if node.nil?
+
+        node.crowbar['state'] = 'applying'
+        node.crowbar['state_owner'] = "#{@bc_name}-#{inst}"
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.